mscore/importove.cpp

```diff
@@ -69,188 +69,218 @@
 
 using namespace Ms;
 
+//---------------------------------------------------------
+//   MeasureToTick
+//---------------------------------------------------------
+
 class MeasureToTick {
-public:
+      int _quarter;
+      Ove::OveSong* _ove;
+
+   public:
       MeasureToTick();
       ~MeasureToTick();
 
-public:
-      void build(OVE::OveSong* ove, int quarter);
+      void build(Ove::OveSong* ove, int quarter);
 
-      int getTick(int measure, int tick_pos);
+      int tick(int measure, int tick_pos);
       static int unitToTick(int unit, int quarter);
 
-      struct TimeTick	{
-            int numerator_;
-            int denominator_;
-            int measure_;
-            int tick_;
-            bool isSymbol_;
+      struct TimeTick {
+            int _numerator;
+            int _denominator;
+            int _measure;
+            int _tick;
+            bool _isSymbol;
 
-            TimeTick():numerator_(4), denominator_(4), measure_(0), tick_(0), isSymbol_(false){}
+            TimeTick():
+                  _numerator(4), _denominator(4), _measure(0), _tick(0), _isSymbol(false) {}
             };
-      QList<TimeTick> getTimeTicks() const;
+      QList<TimeTick> timeTicks() const { return _tts; }
 
-private:
-      int quarter_;
-      OVE::OveSong* ove_;
-      QList<TimeTick> tts_;
+   private:
+      QList<TimeTick> _tts;
       };
 
-int getMeasureTick(int quarter, int num, int den){
-      return quarter * 4 * num / den;
+MeasureToTick::MeasureToTick()
+      {
+      _quarter = 480;
+      _ove = nullptr;
       }
 
-MeasureToTick::MeasureToTick(){
-      quarter_ = 480;
-      ove_ = 0;
+MeasureToTick::~MeasureToTick()
```

Comment:
This can be removed, if there's nothing in it.

Reply:
Done.